src/packages/sdk/src/utils/tryLoadEnvs.ts

```diff
@@ -127,13 +127,21 @@ export function loadEnv(
   if (exists(envPath)) {
     debug(`Environment variables loaded from ${envPath}`)
 
+    const debugEnv = process.env.DEBUG
+
+    // Value needs to be null or undefined, false is truthy
+    // https://github.com/motdotla/dotenv/blob/7301ac9be0b2c766f865bbe24280bf82586d25aa/lib/main.js#L89-L91
+    let enableDebug: true | undefined = undefined
+
+    if (debugEnv && (debugEnv.includes('prisma') || debugEnv === '*')) {
```

Comment:
```suggestion
    if (debugEnv && (debugEnv.startsWith('prisma') || debugEnv === '*')) {
```

To not match if people use the word `prisma` somewhere in their debugging

Reply:
Yes sounds good!